discounts/rust/order-discounts/default/src/tests.rs

```diff
@@ -0,0 +1,20 @@
+use super::*;
+use shopify_function::{run_function_with_input, Result};
+
+#[test]
+fn test_result_contains_no_discounts() -> Result<()> {
+    let result = run_function_with_input(
+        function,
+        r#"
+            {
+                "discountNode": {
+                    "metafield": null
+                }
+            }
+        "#,
+    )?;
+    let expected = 0;
+
+    assert_eq!(result.discounts.len(), expected);
+    Ok(())
+}
```

Comment:
What's the rationale behind going down to 1 test and moving it to a new file?

Reply:
A few reasons:

* Consistency with payment and shipping
* Moving it out means less to copy/paste in tutorials
* One of the tests isn't needed anymore due to use of crate
* General simplicity -- we really just want to show them how to create a test IMO